pallets/asset/src/lib.rs

```diff
@@ -2524,3 +2581,33 @@ impl<T: Config> AssetFnTrait<T::AccountId, T::RuntimeOrigin> for Module<T> {
         Self::add_mandatory_mediators(origin, ticker, mediators.try_into().unwrap_or_default())
     }
 }
+
+pub mod migration {
+    use sp_runtime::runtime_logger::RuntimeLogger;
+
+    use crate::sp_api_hidden_includes_decl_storage::hidden_include::IterableStorageMap;
+    use crate::sp_api_hidden_includes_decl_storage::hidden_include::StorageMap;
+    use crate::sp_api_hidden_includes_decl_storage::hidden_include::StorageValue;
```

Comment:
Please don't import these from a "hidden" crate, they should be available from a public crate.